backend-shared/util/operations/types_test.go

```diff
@@ -100,3 +102,45 @@ var _ = Describe("Testing CreateOperation function.", func() {
 		})
 	})
 })
+
+// TestGenerateOperatorCRName tests the GetOperatorCRName function
+// with different possible db.Operation
+func TestGenerateOperatorCRName(t *testing.T) {
```

Comment:
We are using the Ginkgo framework for unit/e2e tests. I understand it's a utility function and I'm okay with a regular test. I'd let @jgwest decide the convention here

Reply:
changed the tests to Gingko based tests.